src/test/java/uk/gov/hmcts/reform/BaseTest.java

```diff
@@ -0,0 +1,66 @@
+package uk.gov.hmcts.reform;
+
+import uk.gov.hmcts.reform.cpo.domain.CasePaymentOrder;
+import uk.gov.hmcts.reform.cpo.payload.UpdateCasePaymentOrderRequest;
+
+import java.time.LocalDateTime;
+import java.time.Month;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+public interface BaseTest<T> {
+
```

Comment:
The generic was removed  from the interface parameter as it only used for createInitialValuesList()..  createInitialValuesList() method signature can have the generic parameter.   I made this change in the my  PR.

Reply:
Resolved in 94d7cc77493c1245220ac584184bb0099a9c3904